README.md

```diff
@@ -84,552 +84,284 @@ We conducted a comprehensive evaluation of Qwen2.5-Omni, which demonstrates stro
-    <td class="tg-0lax"><strong>0.328</strong>|<strong>0.162</strong>|<strong>0.090</strong>|0.055|0.127|<strong>0.225</strong></td>
-  </tr>
-  <tr>
-    <td class="tg-9j4x" colspan="3">Audio Reasoning</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax" rowspan="3">MMAU<br>Sound | Music | Speech | Avg</td>
-    <td class="tg-0lax">Gemini-Pro-V1.5</td>
-    <td class="tg-0lax">56.75|49.40|58.55|54.90</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Qwen2-Audio</td>
-    <td class="tg-0lax">54.95|50.98|42.04|49.20</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Qwen2.5-Omni-7B</td>
-    <td class="tg-0lax"><strong>67.87|69.16|59.76|65.60</strong></td>
-  </tr>
-  <tr>
-    <td class="tg-9j4x" colspan="3">Voice Chatting</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax" rowspan="8">VoiceBench<br>AlpacaEval | CommonEval | SD-QA | MMSU</td>
-    <td class="tg-0lax">Ultravox-v0.4.1-LLaMA-3.1-8B</td>
-    <td class="tg-0lax"><strong>4.55</strong>|3.90|53.35|47.17</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">MERaLiON</td>
-    <td class="tg-0lax">4.50|3.77|55.06|34.95</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Megrez-3B-Omni</td>
-    <td class="tg-0lax">3.50|2.95|25.95|27.03</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Lyra-Base</td>
-    <td class="tg-0lax">3.85|3.50|38.25|49.74</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">MiniCPM-o</td>
-    <td class="tg-0lax">4.42|<strong>4.15</strong>|50.72|54.78</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Baichuan-Omni-1.5</td>
-    <td class="tg-0lax">4.50|4.05|43.40|57.25</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Qwen2-Audio</td>
-    <td class="tg-0lax">3.74|3.43|35.71|35.72</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Qwen2.5-Omni-7B</td>
-    <td class="tg-0lax">4.49|3.93|<strong>55.71</strong>|<strong>61.32</strong></td>
-  </tr>
-  <tr>
-    <td class="tg-0lax" rowspan="8">VoiceBench<br>OpenBookQA | IFEval | AdvBench | Avg</td>
-    <td class="tg-0lax">Ultravox-v0.4.1-LLaMA-3.1-8B</td>
-    <td class="tg-0lax">65.27|<strong>66.88</strong>|98.46|71.45</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">MERaLiON</td>
-    <td class="tg-0lax">27.23|62.93|94.81|62.91</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Megrez-3B-Omni</td>
-    <td class="tg-0lax">28.35|25.71|87.69|46.25</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Lyra-Base</td>
-    <td class="tg-0lax">72.75|36.28|59.62|57.66</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">MiniCPM-o</td>
-    <td class="tg-0lax">78.02|49.25|97.69|71.69</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Baichuan-Omni-1.5</td>
-    <td class="tg-0lax">74.51|54.54|97.31|71.14</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Qwen2-Audio</td>
-    <td class="tg-0lax">49.45|26.33|96.73|55.35</td>
-  </tr>
-  <tr>
-    <td class="tg-0lax">Qwen2.5-Omni-7B</td>
-    <td class="tg-0lax"><strong>81.10</strong>|52.87|<strong>99.42</strong>|<strong>74.12</strong></td>
-  </tr>
-</tbody></table>
 </details>
 
 <details>
 <summary>Image -> Text</summary>
 
```

Comment:
I considered grouping these as is done in paper instead of how it was in markdown but I think it's pretty self explanatory as is.
<img width="563" alt="Screenshot 2025-04-08 at 23 37 06" src="https://github.com/user-attachments/assets/00602815-1318-425d-9816-e276b26a4ae7" />